process-document/src/main/resources/config/liquibase/changelog/20220920-create-link-process-changelog.xml

```diff
@@ -11,4 +12,13 @@
             </column>
         </createTable>
     </changeSet>
+
+    <changeSet author="Ritense" id="2">
+        <addColumn tableName="document_definition_process_link">
+            <column name="link_type" type="VARCHAR(64)">
+                <constraints nullable="false"/>
```

Comment:
will this not break for existing records?

Reply:
Changed it